src/main/resources/application.yaml

```diff
@@ -178,12 +178,6 @@ envelope-access:
      - jurisdiction: BULKSCAN
```

Comment:
Do we need bulk scan and SSCS ? Can we not completely get rid of this config ?

Reply:
bulkscan is a test service. Whether SSCS uses the read endpoint I do not know.
Just removing the services that were there to enable the retry job.